entname/entname.go

```diff
@@ -224,25 +315,25 @@ func (qn *CommonTrafficQueueNames) String() string {
 }
 
 // CommonTrafficQueues returns the vendors-specific names of common traffic
-// class queues. See the definition of common queues here:
-// https://github.com/openconfig/entity-naming/blob/main/README.md#traffic-queues
+// class queues. See the forwarding group definitions here:
+// https://github.com/openconfig/entity-naming/blob/main/README.md#qos-forwarding-groups
 func CommonTrafficQueues(dev *DeviceParams) (*CommonTrafficQueueNames, error) {
```

Comment:
Is this function still needed? If so I find the doc comment to be unclear since the link doesn't talk about "common traffic class queues" and how they're different from forwarding groups.

Reply:
I meant to mark it deprecated, which is now done. I am leaving it around until I've migrated all callers to the CommonQoSQueues function.